index.html

```diff
@@ -4,6 +4,8 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+<h1>Andrea Ceballos </h1>
```

Comment:
- Remove trailing whitespace, after your name.

All html inside the body tag needs to be indented by one tab level:
```html
<body>
  <h1>...</h1>
</body>
```